Coalition.rte/Devices/Shields/CombatShield/CombatShield.lua

```diff
@@ -0,0 +1,66 @@
+function Coalition_AttachCombatShield(self)
```

Comment:
Can do local function and then make it a self variable in create to avoid (presumably) cluttering up global namespace. Not a big deal though, since it's snake prefixed with the rte

Reply:
Nvm, it's used in pie slice functions so it's not doable.